vent/core/rq_worker/file_watch.py

```diff
@@ -54,13 +55,24 @@ def file_queue(path, template_path="/vent/"):
             if config.has_option(section, 'settings'):
                 try:
                     options_dict = json.loads(config.get(section, 'settings'))
-                    for option in options_dict:
-                        if option == 'ext_types':
-                            ext_types = options_dict[option].split(',')
-                            for ext_type in ext_types:
-                                if path.endswith(ext_type):
-                                    images.append(image_name)
-                                    configs[image_name] = {}
+                    process_file = False
+                    # check if this tool should look at files in base dir
+                    if 'process_base' in options_dict:
+                        if (options_dict['process_base'] == 'yes' and
+                                directory == '/files'):
+                            process_file = True
+                    # check if this tool should look at subdirs created by
+                    # other tools' output
+                    if 'process_from_tool' in options_dict:
+                        for tool in options_dict['process_from_tool'].split(','):
+                            if tool.replace(' ', '-') in directory:
```

Comment:
It gets set in the vent.template for the tool (which then means it gets put in the manifest upon adding the tool). I made it so that every tool in process_from_tool is based on the link_name of the tool which it is looking at. The tools themselves are responsible for creating directories named after themselves so that other tools can view them. For example, the I changed pcap node splitter to create a directory named after itself. I made the directory with the name of the tool with any space replaced with -, but I can change that to have spaces in the directory name if you think that's better.